### ISSUE TITLE
br.i128 split in 2 branches, which is not compliant with BB.

### ISSUE THREAD

**Issue by nbp** [CraneStation/cranelift#1023]
```rust
    narrow.legalize(
        def!(brnz.I128(x, ebb, vararg)),
        vec![
            def!((xl, xh) = isplit(x)),
            def!(brnz(xl, ebb, vararg)),
            def!(brnz(xh, ebb, vararg)),
        ],
    );
```

When replacing one `brnz` instruction by 2 others, this causes issues with basic block assumption that we can only have a single conditional branch in a basic block.

This is blocking #1008 work.
(cc @bnjbvr , @bjorn3 )

**Comment by nbp** [comment]
For information, I managed to add support for block to the meta crate, and fix this issue. I will create the PR on Monday.

**Comment by nbp** [comment]
Fixed by #1034 .
